src/es/hackstore/src/eu/kanade/tachiyomi/animeextension/es/hackstore/Hackstore.kt

```diff
@@ -195,22 +197,35 @@ class Hackstore : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
     override fun episodeFromElement(element: Element): SEpisode { throw UnsupportedOperationException() }
 
     // ============================ Video Links =============================
+    private fun extractUrlFromDonFunction(fullUrl: String): String {
+        val client = OkHttpClient()
+
+        val request = Request.Builder().url(fullUrl).build()
+        val response = client.newCall(request).execute()
```

Comment:
```suggestion
        val response = client.newCall(GET(fullUrl, headers)).execute()
```
You should use the functions provided in `eu.kanade.tachiyomi.network` for making requests